package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "synthetics-recorder",
-  "version": "0.0.1-beta.6",
+  "version": "0.0.1-notarization.3",
```

Comment:
is this the expected version?

Reply:
Good catch, I bumped the version during the testing, will revert it 👍